tests/manage/z_cluster/cluster_expansion/test_add_node_crashcollector.py

```diff
@@ -0,0 +1,136 @@
+import logging
+
+from ocs_ci.framework.testlib import tier2, ignore_leftovers, ManageTest, bugzilla
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
+from ocs_ci.ocs.resources.pod import get_pod_obj, get_all_pods
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import OPENSHIFT_STORAGE_NAMESPACE
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_pod_node, get_mgr_pods
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@ignore_leftovers
+@bugzilla("1898808")
```

Comment:
Please also create a test case in polarion if not done already and add the id .
@pytest.mark.polarion_id("")

Reply:
done